src/main/kotlin/com/openlattice/IdConstants.kt

```diff
@@ -21,50 +21,51 @@
 
 package com.openlattice
 
-import java.util.UUID
+import java.util.*
```

Comment:
Is this necessary?

Reply:
It's just the auto-formatter. It seems to prefer doing .* on java.util but I'm not picky about one or the other. If anything I prefer the .UUID version